test/DynamoCoreTests/SerializationTests.cs

```diff
@@ -782,7 +782,7 @@ public void InPortDescriptionDeserilizationTest()
             Assert.AreEqual(ztNode.InPorts.First().ToolTip, "1st point of arc\n\nPoint");
 
             var nodeModelNode = this.CurrentDynamoModel.CurrentWorkspace.Nodes.Where(x => x.GUID == new Guid("c848cc3cb24a477f8248e53fc9304cc1")).First();
-            Assert.AreEqual(nodeModelNode.InPorts.First().ToolTip, "List of colors to include in the range");
+            Assert.AreEqual(nodeModelNode.InPorts.First().ToolTip, "List of colors to include in the range (disabled)");
```

Comment:
Why is this changed?

Reply:
Is this for fixing the unrelated failing test? https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/17216/testReport/Dynamo.Tests/SerializationTests/InPortDescriptionDeserilizationTest/

Reply:
I don't think this is the expected fix here @dnenov , need to dig why and when was the tooltip changed. Also I think the expected and actual variables are inverted in the test.